mapzen-places-api/src/main/java/com/mapzen/places/api/internal/PeliasFeatureToPlaceConverter.java

```diff
@@ -0,0 +1,60 @@
+package com.mapzen.places.api.internal;
+
+import com.mapzen.pelias.gson.Feature;
+import com.mapzen.places.api.LatLng;
+import com.mapzen.places.api.LatLngBounds;
+import com.mapzen.places.api.Place;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Handles converting Pelias {@link Feature} objects into {@link Place} objects for
+ * {@link PeliasCallbackHandler}.
+ */
+class PeliasFeatureToPlaceConverter {
+
+  private PeliasLayerToPlaceTypeConverter layerConverter;
+
+  /**
+   * Constructor.
+   */
+  PeliasFeatureToPlaceConverter() {
+    layerConverter = new PeliasLayerToPlaceTypeConverter();
+  }
+
+  /**
+   * Construct a {@link Place} object from the given feature.
+   * @param feature
+   * @return
+   */
+  Place getFetchedPlace(Feature feature) {
+    //TODO: fill in missing values
+    final CharSequence address = feature.properties.label;
+    final String id = feature.properties.id;
+    final LatLng latLng = new LatLng(feature.geometry.coordinates.get(1),
+        feature.geometry.coordinates.get(0));
+    final Locale locale = Locale.US;
+    final CharSequence name = feature.properties.name;
+    final List<Integer> placeTypes = new ArrayList<>();
+    if (feature.properties.layer != null) {
+      placeTypes.addAll(layerConverter.getPlaceTypes(feature.properties.layer));
+    }
+    final LatLngBounds viewport = new LatLngBounds(latLng, latLng);
```

Comment:
Good call. Looks like we had logic to do this in `PlaceAutocompletePresenter`...I've extracted this logic into a new class and am now using it in both places to generate a bounding box given a point.